pkg/handlers/webhook/webhook.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2018 Bitnami
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package webhook
+
+import (
+	"fmt"
+	"log"
+	"os"
+
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/bitnami-labs/kubewatch/config"
+	kbEvent "github.com/bitnami-labs/kubewatch/pkg/event"
+)
+
+var webhookErrMsg = `
+%s
+
+You need to set Webhook url
+using "--url/-u" or using environment variables:
+
+export KW_WEBHOOK_URL=webhook_url
+
+Command line flags will override environment variables
+
+`
+
+// Webhook handler implements handler.Handler interface,
+// Notify event to Webhook channel
+type Webhook struct {
+	Url      string
+}
+
+type WebhookMessage struct {
+	Text         string                         `json:"text"`
+}
+
+// Init prepares Webhook configuration
+func (m *Webhook) Init(c *config.Config) error {
```

Comment:
nit: suggest using `w` instead of `m` (likely copypasta from other handler?)

Reply:
I'm using the same syntax that it's used on other handlers:
https://github.com/bitnami-labs/kubewatch/blob/80dafe843fe0db67e34e1b54ad7c674b4700e095/pkg/handlers/mattermost/mattermost.go#L75